src/tests/ftest/util/daos_core_base.py

```diff
@@ -179,10 +164,31 @@ def run_subtest(self):
                 self.fail(
                     "{0} failed with return code={1}.\n".format(
                         job_str, result.result.exit_status))
+        finally:
+            # Copy any remote cmocka files back to this host
+            cmocka_result_file = os.path.join(cmocka_dir, "*_cmocka_results.xml")
+            host_list = []
+            for host in self.hostlist_clients:
+                if check_file_exists(host, cmocka_result_file):
+                    host_list.append(host)
+            if host_list != []:
```

Comment:
(pylint-use-implicit-booleaness-not-comparison) 'host_list != []' can be simplified to 'host_list' as an empty sequence is falsey